### PR TITLE
Added spanish layout json for the vkeyboard

### PR DESCRIPTION
**Pull request by schoolsplay**
Maintainer merge checklist
* [ ] Title is descriptive/clear for inclusion in release notes.
* [ ] Applied a `Component: xxx` label.
* [ ] Applied the `api-deprecation` or `api-break` label.
* [ ] Applied the `release-highlight` label to be highlighted in release notes.
* [ ] Added to the milestone version it was merged into.
* [ ] **Unittests** are included in PR.
* [ ] Properly documented, including `versionadded`, `versionchanged` as needed.

### CODE REVIEW

**Comment by kuzeyron** [comment]
Remember to change the title and description.

File: kivy/data/keyboards/es_ES.json [hunk 1]
```diff
@@ -0,0 +1,97 @@
+{
+    "title": "fr_CH",
```
Comment (NomadDemon) [inline kivy/data/keyboards/es_ES.json:2]: FR?

**Review by akshayaurora — APPROVED** [review]

**Comment by misl6** [comment]
@NomadDemon are you a native Spanish speaker and can confirm the keyboard layout looks correct?

**Comment by Julian-O** [comment]
The request for native Spanish speakers to review would be easier if you posted an image of the keyboard we could run past our social media friends.

**Comment by schoolsplay** [comment]
Sure, here is the keyboard layout in action in some test code.



01662b56-8527-4329-b5d9-4d4bb22bf573)

**Review by akshayaurora — APPROVED** [review]

**Comment by akshayaurora** [comment]
I think we should accept this pr as is for now. If some other Spanish layout finds an issue later on we can get the appropriate changes in at that time.

LGTM!

**Comment by akshayaurora** [comment]
I did a correctness check in that this layout works for me shows up as in the images posted above in my local testing.

**Comment by akshayaurora** [comment]
@schoolsplay Thanks 👍

**Comment by Julian-O** [comment]
I compared the image to other soft keyboards for Spanish, and this seems to be a rather unconventional approach.

It is missing lower case `ü`.
`ñ` is normally on the main keyboard, to the right of the `l`.
`á`, `é`, `í``, `ó` and `ú` are normally entered as two strokes the base letter followed by an accent key.

**Comment by akshayaurora** [comment]
@Julian-O thanks.
Might be unconventional to us but let's wait for some Spanish layout users if they feel this is wrong/ not accurate. We can always add a variants of `es_ES` `es1_ES`... ? None of these are default so it should not be much of a issue.

**Comment by ElRafeta697** [comment]
@akshayaurora as a Spaniard that I am, I agree with @Julian-O. Here I give you a conventional Spanish keyboard in order to show you how it should look (not only the letters, but also the distribution of symbols). The "Ç" letter is an exclusive Catalonian/Valencian letter. It's optional, but is't preferible that it be an independent key (I'm Valencian btw)

61PXDscB52L _AC_SX425_

**Comment by akshayaurora** [comment]
@schoolsplay would you please address the issues mentioned above.

**Comment by schoolsplay** [comment]
I will just run it by our spanish sales people but if this is a regular layout for spanish I can change it.

**Comment by schoolsplay** [comment]
Created a new pull request with a updated layout created with our Spanish colleagues. 
https://github.com/kivy/kivy/pull/8532#issue-2058400180

**Pull request merged**